docs/components/modeler/web-modeler/import-diagram.md

```diff
@@ -0,0 +1,31 @@
+---
+id: import-diagram
+title: Import diagram
+description: "You can import a BPMN diagram at any time with Web Modeler."
+---
+
+You can import a BPMN diagram at any time with Web Modeler.
+
+There are different ways how you can achieve that:
+
+1. In a Project, click **New** and choose **Upload Files** and then select the files from your machine
+![import diagram](img/import-diagram/web-modeler-upload-file-menu-item.png)
+![import diagram](img/import-diagram/web-modeler-upload-file-choose.png)
+![import diagram](img/import-diagram/web-modeler-upload-file-completed.png)
+
+2. In a Project, drag one file from your machine and drop it
+![import diagram](img/import-diagram/web-modeler-project-drag-and-drop.png)
+
+3. In a Diagram, open the breadcrumb menu and choose **Replace via Upload** and then select a file from your machine
+![import diagram](img/import-diagram/web-modeler-replace-via-upload-menu-item.png)
+![import diagram](img/import-diagram/web-modeler-replace-via-upload-choose.png)
+:::note
+The content of the Diagram will be replaced with the content of the file
+:::
+
+4. In a Diagram, drag one file from your machine and drop it onto the canvas
+![import diagram](img/import-diagram/web-modeler-diagram-replace-via-drag-and-drop.png)
+:::note
+The content of the Diagram will be replaced with the content of the file
+:::
```

Comment:
To me displaying a second note with the exact same content as one bullet point higher is confusing and looks more like a error than intention. Could we either merge the two notes or phrase the second one a little different?